portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -103,7 +102,20 @@ function populateDropdown(list, ID) {
   });
 }
 
+// inserts a functioning searchbar into the navigation bar of a page.
```

Comment:
Could you turn this into proper JSDoc? https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Reply:
I transferred this comment to relevant repository.